minecraft_script/compiler/compile_interpreter.py

```
@@ -296,27 +335,38 @@ def visit_AsyncWhileLoopNode(self, node, context: CompileContext) -> CompileResu
         selector_id = generate_uuid()
         out: CompileResult = self.visit(node.get_body(), loop_context)
         condition: mcs_type = self.visit(node.get_condition(), condition_context).get_value()
-        loop_init_cmd = f"function {self.datapack_id}:{condition_context.mcfunction_name}"
+        loop_init_cmd = self.version.render(
+            "control.async_while.init",
+            conditionPath=condition_context.mcfunction_name,
+        )
         loop_init_cmd = add_comment(loop_init_cmd, f"Initialize async while loop:")
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)